web_src/js/features/codeeditor.js

```diff
@@ -13,7 +13,6 @@ const baseOptions = {
   overviewRulerLanes: 0,
   renderLineHighlight: 'all',
   renderLineHighlightOnlyWhenFocus: true,
-  renderWhitespace: 'none',
```

Comment:
Curious, why it contains an unrelated change .....

Reply:
It was due to a mixup with vscode, and was overwritten by a force push to the 1.19 branch. I am attempting to cherry pick it back into the correct branch.